core_lib/web_helpers/flask/user_auth_middleware.py

```diff
@@ -0,0 +1,15 @@
+from flask import Request
+from core_lib.session.security_handler import SecurityHandler
+
+
+class UserAuthMiddleware():
+
+    def __init__(self, app):
+        self.app = app
+
+    def __call__(self, environ, start_response):
+        request = Request(environ)
+        if request.cookies.get('auth_token'):

```

Comment:
What if the user wants a different name for the cookie? 
`auth_token` is hardcoded here, 
is it poss the cookie name in the contractor